03-Security-In-CICD/Readme.md

```diff
@@ -154,8 +156,7 @@ next to it so that it gets masked.
 (You may skip this step in a first-pass exploration of CICD integration of SVTs.) 
 This feature enables you to set up online policies. 
 This enables the CICD extension to use org-specific policies. 
-<!-- #TODO# Clarify. Need IT-specific instructions too! Or it should just work for IT. --> 
-
+To use org-specific policies, you can get your org-specific url from AzSDKSettings.json file under 'OnlinePolicyStoreUrl' parameter. 
```

Comment:
we should provide the location of setting file..how would user know the location of AzSDKSettings.json